src/components/Dashboard/EditUserData/index.jsx

```diff
@@ -25,9 +27,13 @@ function EditUserData() {
   }, [form, image]);
   useEffect(() => {
     (async () => {
-      const { payload } = await dispatch(getUserProfile(userId));
+      const payload = await dispatch(getUserProfile(userId)).unwrap();
       form.setFieldsValue(payload.data);
-      setOldImage(payload.data.image);
+      if (payload.data.image) {
+        setOldImage(payload.data.image);
+      } else {
+        setOldImage(DEFAULT_BANNER);
+      }
```

Comment:
```suggestion
    const { data } = await dispatch(getUserProfile(userId)).unwrap();
    setOldImage(data.image ?? DEFAULT_BANNER );
```
instead of `if /else`